.travis.yml

```diff
@@ -9,6 +9,21 @@ php:
   - 7.1
   - hhvm
 
-before_script: composer install
+# This triggers builds to run on the new TravisCI infrastructure.
+# See: http://docs.travis-ci.com/user/workers/container-based-infrastructure/
+sudo: false
+
+## Cache composer
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+matrix:
+  include:
+    - php: 5.3
+      env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
+
+before_script:
+  - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-dist
```

Comment:
`--no-interaction` is not a default on composer, however it is already applied by travis, by setting an environment variable.